### PR TITLE
Create lookup+score

### PR DESCRIPTION
**Pull request by finnagin**

### CODE REVIEW

**Comment by cbizon** [comment]
Do we have a formal way of designating these combo operations?  I checked lookup (which is equivalent to fill+bind+complete) and didn't see anything.  But in terms of the workflow runner, aren't we going to need that?

**Comment by finnagin** [comment]
I'm not aware of anything. Should we try to come up with one? @patrickkwang @dkoslicki @edeutsch thoughts?

**Comment by edeutsch** [comment]
I am not aware of anything.

**Comment by dkoslicki** [comment]
I like the idea of having a flag that indicates this, similar to `unique`. No strong feelings on what it should be called though (`combo_operation`, `meta_operation`, ?)

**Comment by finnagin** [comment]
How about `composite_operation`?

File: operations/lookup_and_score.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+- input: fill_and_bind/messages/01_qgraph.json
+  output: fill_and_bind/messages/05_scored_results.json
+input_requirements: []
+output_guarantees:
+- results complete
+allowed_changes:
+- add knodes
+- add kedges
+- add results
+- add properties to results
```
Comment (patrickkwang) [inline operations/lookup_and_score.yml:16]:
```suggestion
```
This is subsumed by `add results`.

**Review by patrickkwang — APPROVED** [review]

**Pull request merged**